app/app/settings.py

```diff
@@ -50,6 +50,16 @@
     'django.contrib.auth.middleware.AuthenticationMiddleware',
     'django.contrib.messages.middleware.MessageMiddleware',
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+	'app.corsMiddleware.cors_middleware',
+	'corsheaders.middleware.CorsMiddleware'
```

Comment:
@routrohan this needs to be in requirements.txt

Reply:
I followed [this](https://stackoverflow.com/questions/44037474/cors-error-while-consuming-calling-rest-api-with-react) and it worked for me without any issues
Just add `pip install django-cors-headers` to the `requirements.txt` and we're good to go